R/applyFilt.R

```diff
@@ -2081,10 +2151,23 @@ pmartR_filter_worker <- function (filter_object, omicsData) {
     # remove any emeta molecules from e_meta and e_data #
     if (!is.null(filter_object$e_meta_remove)) {
 
+      valid_names <- omicsData$e_meta[, which(names(omicsData$e_meta) == emeta_cname)]
+      
+      # Warn if trying to remove biomolecules which are no longer in the data
```

Comment:
Should this be commented out?  Looks to be in the correct spot.

Reply:
It was just giving a lot of extra warnings when running the existing tests, so I commented it out.

Reply:
Ah yea I see them too, since the filter worker filters out the e_data first *as well as any corresponding stuff in e_meta* and then it's all already gone once it gets to removing e_meta.  Hmmm...I'll think a bit about what to do here...